contents/docs/integrate/send-events/_snippets/send-events-react-native.mdx

```diff
@@ -64,15 +44,16 @@ export function App() {
 }
 ```
 
-#### Tracking Screen views and touches with `react-native-navigation`:
+#### With `react-native-navigation` and autocapture:
```

Comment:
@marandaneto I didn't really understand what we're doing in this section. Can you please have a read through a let me know if it's okay / can be improved?

Reply:
It is just another navigation library and how to set up the autocapture, very similar to the section above doing the very same thing, what you did not understand? the text or the code snippet? can you explain?